### PR TITLE
Remove fomantic grid module

### PR DESCRIPTION
**Pull request by silverwind**
Removed the grid module and moved the used parts it into our own CSS, eliminating around 75% unused CSS in turn.

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: web_src/css/modules/grid.css [hunk 1]
```diff
@@ -0,0 +1,495 @@
+.ui.grid {
```
Comment (techknowlogick) [inline web_src/css/modules/grid.css:4]: can you add a note to this file about the rules sourced from fomantic/semantic?

Comment (silverwind) [reply web_src/css/modules/grid.css:4]: Many of the `module` files are such "extractions", but I guess I can add notes where appropriate.

**Review by wxiaoguang — APPROVED** [review]
Don't know how you managed to pick these styles out ..... Over all LGTM.

**Comment by silverwind** [comment]
> Don't know how you managed to pick these styles out ..... Over all LGTM.

Through tedious searching the classes in the code :)

**Comment by silverwind** [comment]
Draft until I add comments.

**Comment by silverwind** [comment]
Comments added, only "grid" and "message" are currently pure copy-paste from fomantic, more likely to follow.

**Pull request merged**